### DISCUSSION TITLE
Position Open Shell start button on top of W11 Start button

### DISCUSSION THREAD

**Discussion by perdrix52** [Open-Shell/Open-Shell-Menu#1212] [Q&A] [answered]
If I configure the location of the W11 Start button center screen (W11 default), is there any way to position the Open Shell Start button at the same location?   I am running ExplorerPatcher and latest release build of OpenShell.

Thanks
David

**Comment by bonzibudd** [comment, marked as answer]
Try the latest release: https://github.com/Open-Shell/Open-Shell-Menu/releases

**Comment by perdrix52** [comment]
Thank you

**Reply by FoldZero** [reply]
What is the setting in Openshell to do this?

- Never mind in installed the latest release and its in the centre from the start.
- Nice Work!
